tests/framework/extensions/projects/listprojects.go

```diff
@@ -0,0 +1,22 @@
+package projects
+
+import (
+	"sort"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+)
+
+func ListProjectNames(client *rancher.Client, clusterID string) ([]string, error) {
```

Comment:
Public functions need their own godoc comment explanation. Could we add an explanation? Also, what's the difference between GetProjectList and this function? Can't we sort this in that function if it's only sorting?

Reply:
With the GetProjectList function we get a collection of projects in a cluster. ListProjectNames, returns the sorted names of just ProjectNames and returns a sorted list